src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const treeItems = document.querySelectorAll('.tree li');
+
+treeItems.forEach((item) => {
+  const childUl = item.querySelector('ul');
+
+  if (childUl) {
+    const span = document.createElement('span');
+
+    span.textContent = item.firstChild.textContent;
+    item.firstChild.replaceWith(span);
+
+    // childUl.style.display = 'none';
```

Comment:
The line to hide the child list initially is commented out. If the intended behavior is to start with the child lists hidden, this line should be uncommented.